LoopKit/CarbKit/HKQuantitySample+CarbKit.swift

```diff
@@ -10,21 +10,30 @@ import HealthKit
 
 
 let MetadataKeyAbsorptionTimeMinutes = "com.loudnate.CarbKit.HKMetadataKey.AbsorptionTimeMinutes"
```

Comment:
If we change these now, it would break existing installations of DIY. @ps2?

Reply:
I think since the cache is being used as the primary source of truth right now, it probably wouldn't break DIY, but it may break other apps that expect it to be named that, like Tidepool Mobile. 
I would say keep it as is, unless we have some compelling reason to change it, and have a plan for other apps.